src/main/kotlin/pt/up/fe/ni/website/backend/service/activity/ProjectService.kt

```diff
@@ -74,4 +79,24 @@ class ProjectService(
         project.isArchived = false
         return repository.save(project)
     }
+
+    fun addHallOfFameMemberById(idProject: Long, idAccount: Long): Project {
+        val project = getProjectById(idProject)
+        val account = accountService.getAccountById(idAccount)
+        project.hallOfFame.add(account)
+        return repository.save(project)
+    }
+
+    fun removeHallOfFameMemberById(idProject: Long, idAccount: Long): Project {
+        val project = getProjectById(idProject)
+        if (!accountService.doesAccountExist(idAccount)) {
+            throw NoSuchElementException(
+                ErrorMessages.accountNotFound(
+                    idAccount
+                )
+            )
+        }
+        project.hallOfFame.removeIf { it.id == idAccount }
```

Comment:
This is the line the code coverage bot warns about, I guess changing its behavior would fix it, but it would do the same thing anyway. I think there is no need to update it.

Reply:
solved bu changing codecov config